### ISSUE TITLE
Wont open again

### ISSUE THREAD

**Issue by Boggy23** [MCMi460/NSO-RPC#117]
After opeing it for the first time, it has not opened again since, and ive reset my computer and downloaded it again multiple times, yet it still doesnt open up, but the program appears as running on task manager, yet it still doesnt appear on the desktop

**Comment by HotaruBlaze** [comment]
Did you enable autostart and minimise to tray?

**Comment by Boggy23** [comment]
Yes, and i even went on documents and deleted the file for it, it opened again once i redownloaded it again, and it still wont open again

**Comment by HotaruBlaze** [comment]
Can you try deleting the folder, killing the app via task manager and launching it again.

**Comment by Boggy23** [comment]
I did it, and it made me sign in again, but when i go to add the status, it says it gotta resart. and when i press ok, it doesnt open again

**Comment by HotaruBlaze** [comment]
yeah so currently on stable, the app wont attempt to autostart and is actually asking for you to start it. This only changes recently with PR #115 and isnt part of the build your using.

**Comment by Boggy23** [comment]
Ok, thank you. i asked my friend to show me how she got it and i think we got it figured out. thanks for all the help

**Comment by MCMi460** [comment]
This seems like it was in your system tray, from what I'm reading?
I'm glad it got worked out. Closing issue as problem is solved.
